lib/migrations/20190418160820_student_additional_details.js

```diff
@@ -0,0 +1,20 @@
+exports.up = function(knex, Promise) {
+    return knex.schema.table('students', function(t) {
```

Comment:
The columns `percentageIn10th ` should be a string because we don't know if it is a % or a CGPA. The user can also write "9.7 CGPA" or "9.7%".

For `mathMarksIn10th `, `percentageIn12th` and `mathMarksIn12th` should be decimal fields.

Reply:
we did this task